src/utils.js

```diff
@@ -116,7 +116,7 @@ function matches(textToMatch, matcher) {
   if (matcher instanceof RegExp) {
     return matcher.test(textToMatch)
   } else {
-    return textToMatch.toLowerCase().includes(String(matcher).toLowerCase())
+    return textToMatch.includes(matcher)
```

Comment:
I would keep `matcher` wrapped in `String()`. This allows to pass non string stuff (such as numbers) to `.toHaveTextContent`, and removing it as part of this change adds more surface for tests to break for issues not related with what you're trying to fix (case sensitiveness).

```suggestion
    return textToMatch.includes(String(matcher))
```

Reply:
It makes sense. I'll do it.